modules/nw-allocate-load-balancers-to-subnets-procedure.adoc

```diff
@@ -0,0 +1,124 @@
+// Module included in the following assemblies:
+//
+// *networking/allocating-load-balancers.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-allocating-load-balancers-to-subnets-procedure_{context}"]
+= Specifying AWS Subnets for OpenShift Ingress Load Balancers at Installation
+
+Perform the following steps to allocate API and Ingress load balancers to specific subnets.
+
+.Prerequisites
+
+Before you begin, ensure you have:
+
+* An existing AWS virtual private cloud (VPC).
+
+* Pre-configured AWS subnets intended for use by the OpenShift cluster, with the following considerations:
+
+** A list of their subnet IDs (for example, `subnet-0123456789abcdef0`). These IDs will be used in the `install-config.yaml` file.
+
+** It is highly recommended that you use subnets spanning at least two availability zones (AZs) for high availability of load balancers and other critical components like control planes.
+
+** Sufficient available IP addresses within these subnets for all assigned roles.
+
+** The AWS configuration for these subnets, including Network ACLs and Security Groups, must permit necessary traffic for all roles assigned to them. For subnets hosting an Ingress Controller, this typically includes TCP ports 80 and 443 from desired sources.
```

Comment:
```suggestion
** The AWS configuration for these subnets, including network ACLs and security groups, must permit necessary traffic for all roles assigned to them. For subnets hosting an ingress controller, this typically includes TCP ports 80 and 443 from required sources.
```
use of the word "desired": See [guidance](https://www.ibm.com/docs/en/ibm-style?topic=word-usage)